File: CursoJava/src/Exer/EstruturaRepetitiva/Exer01.java
package Exer.EstruturaRepetitiva;

import java.util.Locale;
import java.util.Scanner;

public class Exer01 {

	public static void main(String[] args) {
		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);

		System.out.print("Senha: ");
		int senha = sc.nextInt();

		while (senha != 2002) {
			System.out.println("Senha Invalida.");
			senha = sc.nextInt();
		}

		System.out.println("Acesso Permitido.");

		sc.close();

	}

}

File: CursoCsharpUdemy/exer/udemy/EstruturaRepetitiva/While/Exer01.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace exer.udemy.EstruturaRepetitiva.While
{
    public class Exer01
    {
        static void Main(string[] args)
        {
            Console.Write("Senhaenha: ");
            int pass = int.Parse(Console.ReadLine());

            while (pass != 2002)
            {
                Console.WriteLine("Senha Invalida.");
                Console.WriteLine("---------------");
                Console.Write("Informe novamente: ");
                pass = Console.ReadLine();
            }

            Console.WriteLine("Acesso Permitido.");
        }
    }
}